libraries/AP_Scripting/examples/disarm_on_flip.lua

```diff
@@ -0,0 +1,38 @@
+--Example For disarming the drone when roll or pitch exceeds n degrees below n altitude.
+--used to hand recover quads and to prevent prop strikes in case of low altitude collision
+--Must have a downward facing rangefinder
+--Made by shroxgh
+
+local rotation_downward = 25
+local ALT_THRESH = 250 -- Altitude in which in centimeters 
+local ANGLE_THRESH = 80 -- Roll/Pitch angle threshold in degrees
+

```

Comment:
Not a blocker but normally we print a message to tell the user that the script has been loaded.

gcs:send_text(1, "copter-disarnm-on-flip script loaded")